chroma_core/lib/stratagem.py

```diff
@@ -0,0 +1,171 @@
+import requests
+import settings
+import operator
+import json
+
+from toolz.functoolz import pipe, partial
+from requests.exceptions import ConnectionError
+
+temp_stratagem_measurement = "temp_stratagem_scan"
+stratagem_measurement = "stratagem_scan"
+
+size_distribution_name_table = {
+    "size < 1m": "less_than_1m",
+    "1m <= size < 1g": "greater_than_equal_1m_less_than_1g",
+    "size >= 1g": "greater_than_equal_1g",
+    "size >= 1t": "greater_than_equal_1t",
+}
+
+
+filter_out_other_counter = partial(filter, lambda counter: counter.get("name").lower() != "other")
+flatten = lambda xs: [item for l in xs for item in l]
+
+
+def tuple_to_equals(xs):
+    return "=".join((str(xs[0]), str(xs[1])))
```

Comment:
Couldn't this be a format string?

Reply:
```
"{}={}".format(str(xs[0]), str(xs[1]))
```
?

Reply:
Won't just passing to the format fn convert to a str representation?

Reply:
I tried a few things but not sure i'm following. 
```
>>> "{}={}".format(x)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: tuple index out of range
```

Reply:
`"{}={}".format(*x)`